MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/HomePage2/SearchVC/SearchViewController+Design.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+import SnapKit
+
+extension SearchViewController: DesignProtocol {
+    
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        tableView = UITableView(frame: .zero, style: .grouped)
+        view.addSubview(tableView)
+    }
+    
+    func styleViews() {
+        view.backgroundColor = .white
+        
+        tableView.rowHeight = 142 + 12
```

Comment:
why don't you write 154?

Reply:
[Here](https://github.com/lihter/Movie-App/pull/41/commits/3383b62f0e45eb7e1f5ba5b650a01beaa73387a1#diff-aad6f6394c4bd80772719da5c89e1ef095add811bfc58f7abd070251bcbbeba6R20)